dist/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
@danwebguy Remove dist folder from your repo. I.E add it to .gitignore file

Reply:
ok @tosmak16